packages/server/src/fhir/operations/projectinit.test.ts

```diff
@@ -0,0 +1,209 @@
+        parameter: [
+          {
+            name: 'name',
+            valueString: projectName,
+          },
+          {
+            name: 'owner',
+            valueString: createReference(owner).reference,
+          },
+        ],
+      });
+    expect(res.status).toBe(201);
+
+    const project = res.body as Project;
+    expect(project.id).toBeDefined();
+    expect(isUUID(project.id as string)).toBe(true);
+    expect(project.owner).toEqual(createReference(owner));
+  });
+
+  test('Requires project name', async () => {
+    const superAdminAccessToken = await initTestAuth({ superAdmin: true });
+    expect(superAdminAccessToken).toBeDefined();
+
+    const owner = await createUser({
+      email: randomUUID() + '@example.com',
+      password: 'iaudhbrglkjhabdfligubhaedrg',
+      firstName: 'The',
+      lastName: 'Admin',
+    });
+
+    const res = await request(app)
+      .post(`/fhir/R4/Project/$init`)
+      .set('Authorization', 'Bearer ' + superAdminAccessToken)
+      .set('Content-Type', ContentType.FHIR_JSON)
+      .set('X-Medplum', 'extended')
+      .send({
+        resourceType: 'Parameters',
+        parameter: [
+          {
+            name: 'owner',
+            valueString: createReference(owner).reference,
+          },
+        ],
+      });
+    expect(res.status).toBe(400);
+  });
+
+  test('Requires owner to be User', async () => {
+    const superAdminClientToken = await initTestAuth({ superAdmin: true });
+    expect(superAdminClientToken).toBeDefined();
+
+    const projectName = 'Test Init Project ' + randomUUID();
+    const res = await request(app)
+      .post(`/fhir/R4/Project/$init`)
+      .set('Authorization', 'Bearer ' + superAdminClientToken)
+      .set('Content-Type', ContentType.FHIR_JSON)
+      .set('X-Medplum', 'extended')
+      .send({
+        resourceType: 'Parameters',
+        parameter: [
+          {
+            name: 'name',
+            valueString: projectName,
+          },
+        ],
+      });
+    expect(res.status).toBe(400);
+  });
+
+  test('Requires server User', async () => {
+    const accessToken = await initTestAuth();
+    expect(accessToken).toBeDefined();
+
+    const projectName = 'Test Init Project ' + randomUUID();
+    const owner = await createUser({
+      email: randomUUID() + '@example.com',
+      password: 'iaudhbrglkjhabdfligubhaedrg',
+      firstName: 'Other Project',
+      lastName: 'Member',
+      projectId: randomUUID(),
+    });
+
+    const res = await request(app)
+      .post(`/fhir/R4/Project/$init`)
+      .set('Authorization', 'Bearer ' + accessToken)
+      .set('Content-Type', ContentType.FHIR_JSON)
+      .set('X-Medplum', 'extended')
+      .send({
+        resourceType: 'Parameters',
+        parameter: [
+          {
+            name: 'name',
+            valueString: projectName,
+          },
+          {
+            name: 'owner',
+            valueString: createReference(owner).reference,
```

Comment:
Thoughts on making this `valueReference: createReference(owner)`?  Might be more FHIR-y?